### ISSUE TITLE
Consider otel span to be a transaction of type=message when some message attribute is set

### ISSUE THREAD

**Issue by simitt** [elastic/apm-data#194]
[`TranslateTransaction`](https://github.com/elastic/apm-data/blob/main/input/otlp/traces.go#L230) only considers events that have `message_bus.destination` or `messaging.destination` set to be a transaction of type `messaging`. However, `messaging.system` is a required field according to https://opentelemetry.io/docs/specs/semconv/messaging/messaging-spans/ and should also be considered for deciding the transactions type. 

See https://github.com/elastic/apm-data/blob/main/input/otlp/traces.go#L411-L414

**Comment by carsonip** [comment]
- In `TranslateSpan`, we are currently storing messageSystem in `event.Span.Subtype` and messageOperation in event.Span.Action.
- In `TranslateTransaction`, `messaging.system` and `messaging.operation` are currently set as labels under `default` in the `switch` statement.

Question:
We don't have subtype and action in Transaction. Should we still keep the info in a label, after using it to determine transaction type? In that case the change will look like 
```diff
diff --git a/input/otlp/traces.go b/input/otlp/traces.go
index 94072c7..8b8da65 100644
--- a/input/otlp/traces.go
+++ b/input/otlp/traces.go
@@ -413,6 +413,12 @@ func TranslateTransaction(
                        case "message_bus.destination", semconv.AttributeMessagingDestination:
                                message.QueueName = stringval
                                isMessaging = true
+                       case semconv.AttributeMessagingSystem:
+                               isMessaging = true
+                               modelpb.Labels(event.Labels).Set(k, stringval)
+                       case semconv.AttributeMessagingOperation:
+                               isMessaging = true
+                               modelpb.Labels(event.Labels).Set(k, stringval)
 
                        // rpc.*
                        //
```

**Comment by simitt** [comment]
Yes let's keep the information set as labels. Eventually the information should be mapped to the `messaging.*` field, but that would be a larger undertaking as the current apm model for [`message`](https://github.com/elastic/apm-data/blob/main/model/modeljson/message.pb.json.go#L25) differs quite a bit from otel semantic conventions for [`messaging`](https://opentelemetry.io/docs/specs/semconv/messaging/messaging-spans/), and the apm model doesn't have attributes for `system` and `operations`.

**Comment by carsonip** [comment]
Closes via #204
